packages/compiler/core/checker.ts

```diff
@@ -2489,23 +2632,60 @@ export function createChecker(program: Program): Checker {
     const props: ModelProperty[] = [];
     // copy each property
     for (const prop of walkPropertiesInherited(targetType)) {
-      const newProp = cloneType(prop, {
-        sourceProperty: prop,
-        model: parentModel,
-      });
-      props.push(newProp);
+      props.push(
+        cloneType(prop, {
+          sourceProperty: prop,
+          model: parentModel,
+        })
+      );
     }
     return props;
   }
 
+  /**
+   * Link a computed model property to its model member symbols.
+   * @param containerNode Model Node
+   * @param member New Property
+   * @param mapper Type Mapper.
+   */
+  function linkComputedMember(
```

Comment:
`computed` has me confused with `computed properties` in JS/TS, but I can't come up with a better name so I'm just gonna whine about it a little and offer no practical solution.

Reply:
`linkIndirectMember` maybe?